domain/status/state/types.go

```diff
@@ -203,6 +204,10 @@ type applicationNameUnitCount struct {
 	UnitCount int    `db:"unit_count"`
 }
 
+type modelUUID struct {
+	UUID coremodel.UUID `db:"uuid"`
```

Comment:
We've settled on using `string` rather than core UUID types

Reply:
refactored with thanks ian!